build/BuildNuGets.csx

```diff
@@ -3,17 +3,18 @@ using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
 
-string usage = @"usage: BuildNuGets.csx <binaries-dir> <build-version>";
+string usage = @"usage: BuildNuGets.csx <binaries-dir> <build-version> <output-directory>";
 
-if (Args.Length != 2)
+if (Args.Count() != 3)
 {
     Console.WriteLine(usage);
     Environment.Exit(1);
 }
 
-var binDir = Path.GetFullPath(Args[0]);
+var binDir = Path.GetFullPath(Args[0]).TrimEnd('\\');
```

Comment:
Ironically, I think `/` is fine -- the problem is that the ending `\` appears to be parsed by `nuget.exe` as an escape sequence. When I surround the path in quotes (in case there are spaces) the end quote is regarded as an escaped quote, which makes the quote extend to the end of the line.